zerver/management/commands/register_server.py

```diff
@@ -58,16 +58,10 @@ def handle(self, *args: Any, **options: Any) -> None:
             raise CommandError(
                 "Missing zulip_org_key; run scripts/setup/generate_secrets.py to generate."
             )
-        if settings.PUSH_NOTIFICATION_BOUNCER_URL is None:
-            if settings.DEVELOPMENT:
-                settings.PUSH_NOTIFICATION_BOUNCER_URL = (
-                    settings.EXTERNAL_URI_SCHEME + settings.EXTERNAL_HOST
-                )
-            else:
-                raise CommandError(
-                    "Please uncomment PUSH_NOTIFICATION_BOUNCER_URL "
-                    "in /etc/zulip/settings.py (remove the '#')"
-                )
+        if not settings.ZULIP_SERVICE_PUSH_NOTIFICATIONS:
+            raise CommandError(
+                "Please set ZULIP_SERVICE_PUSH_NOTIFICATIONS to True in /etc/zulip/settings.py"
+            )
```

Comment:
Does this need to check `ZULIP_SERVICES_URL` too?

Reply:
I guess we expect that to have a default value in most configurations, so it may not be important, but I think having some sort of `ZULIP_SERVICES_URL` error message would be good error-handling.

Reply:
Yeah pretty unlikely, since someone would need to explicitly add `ZULIP_SERVICES_URL = None` to their settings - but can't hurt to have an extra check here. Added.